docs/machine-learning/resources/glossary.md

```diff
@@ -75,19 +75,19 @@ A feature extraction scheme for text data. Any sequence of N words turns into a
 
 ## Numerical feature vectors
 
-A feature vector consisting only of numerical values. This is similar to double[].
+A feature vector consisting only of numerical values. This is similar to `double[]`.
 
 ## Pipeline
 
 All of the operations needed to fit a model to a dataset. A pipeline consists of data import, transformation, featurization, and learning steps. Once a pipeline is trained, it turns into a model.
 
 ## Precision
 
-The proportion of true results to positive results.
+In [classification](#classification), the precision for a class is the number of items correctly predicted as belonging to that class divided by the total number of items predicted as belonging to the class.
```

Comment:
Or should I reference binary classification, as I see precision and recall only in [BinaryClassificationMetrics class](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.models.binaryclassificationmetrics?view=ml-dotnet)?